org.lflang.tests/src/org/lflang/tests/Configurators.java

```diff
@@ -83,7 +83,8 @@ public static boolean compatibleWithThreadingOff(TestCategory category) {
             || category == TestCategory.SERIALIZATION
             || category == TestCategory.FEDERATED
             || category == TestCategory.DOCKER_FEDERATED
-            || category == TestCategory.DOCKER;
+            || category == TestCategory.DOCKER
+            || category == TestCategory.ARDUINO; //TODO: Should have a better solution for having Arduino Tests not execute.
```

Comment:
This doesn't look right. Arduino doesn't belong in the list of categories that are not compatible with single-threaded execution.